fulp_modules/features/antagonists/infiltrators/infil_items.dm

```diff
@@ -523,14 +528,16 @@
 	antag_datum = /datum/antagonist/infiltrator_backup
 
 /obj/item/antag_spawner/nuke_ops/infiltrator_backup/attack_self(mob/user)
+	if(used)
+		return
 	var/datum/antagonist/traitor/infiltrator/criminal = user.mind.has_antag_datum(/datum/antagonist/traitor/infiltrator)
 	if(!criminal)
 		return
-	if(!do_after(user, 5 SECONDS))
-		return
 	to_chat(user, span_notice("You activate [src] and wait for confirmation."))
 	var/list/infil_candidates = poll_ghost_candidates("Do you want to play as an infiltrator backup?", ROLE_INFILTRATOR, ROLE_INFILTRATOR, 150, POLL_IGNORE_SYNDICATE)
 	if(LAZYLEN(infil_candidates))
+		if(QDELETED(src) || used)
```

Comment:
isn't || used unnecesary since we're already returning if it's used 7 lines above

Reply:
its necessary here incase the player spams the button and summons 9000000000000000000 pop ups and then ull have 9000000000000000 players clicking yes all at the same time